src/TestSuite/IntegrationTestTrait.php

```diff
@@ -509,11 +502,7 @@ protected function _sendRequest($url, $method, $data = []): void
                 $this->_requestSession->write('Flash', $this->_flashMessages);
             }
             $this->_response = $response;
-        } catch (PhpUnitError $e) {
-            throw $e;
-        } catch (DatabaseException $e) {
-            throw $e;
-        } catch (LogicException $e) {
+        } catch (PhpUnitError | DatabaseException | LogicException $e) {
```

Comment:
`PHPUnit\Framework\Error\Error as PhpUnitError` only catches PHP errors/warning/notices.

`\PHPUnit\Framework\Exception` class or `\PHPUnit\Exception` interface would be better as it catch all PHPUnit asserts too.

Reply:
This is a great approach! Well done.

Reply:
I don't think we want to catch phpunit assertion errors here though.

Reply:
I wanted to say "it should re-throw all PHPUnit asserts".

Currently they are catched in `\Throwable` block and rendered by `ExceptionRenderer`.